node-api-implementer.md

```diff
@@ -0,0 +1,42 @@
+# Maintaining @nodejs/node-api-implementer
+
```

Comment:
```suggestion

The @nodejs/node-api implementer team is used a method for the Node-api team to
community and share information with implmentations of Node-api outside of Node.js
itself.  It will be used to gather feedback on major changes that may affect those
implementations.
```